unittests/auth_tests.cpp

```diff
@@ -17,81 +17,90 @@ using namespace eosio::testing;
+   TESTER chain;
+
+   chain.create_accounts( {"alice"_n, "bob"_n} );
+   chain.produce_block();
 
    /// action not provided from authority
-   BOOST_REQUIRE_THROW( push_reqauth( "alice"_n, {permission_level{"bob"_n, config::active_name}}, { get_private_key("bob"_n, "active") } ), missing_auth_exception);
+   BOOST_REQUIRE_THROW( chain.push_reqauth( "alice"_n, {permission_level{"bob"_n, config::active_name}}, { chain.get_private_key("bob"_n, "active") } ), missing_auth_exception);
 
 } FC_LOG_AND_RETHROW() } /// transfer_test
 
 /**
  *  This test case will attempt to allow one account to transfer on behalf
  *  of another account by updating the active authority.
  */
-BOOST_FIXTURE_TEST_CASE( delegate_auth, validating_tester ) { try {
-   create_accounts( {"alice"_n,"bob"_n});
-   produce_block();
+BOOST_AUTO_TEST_CASE_TEMPLATE( delegate_auth, TESTER, validating_testers ) { try {
+   TESTER chain;
+
+   chain.create_accounts( {"alice"_n,"bob"_n});
+   chain.produce_block();
 
    auto delegated_auth = authority( 1, {},
                           {
                             { .permission = {"bob"_n,config::active_name}, .weight = 1}
                           });
 
-   auto original_auth = control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
+   auto original_auth = chain.control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
    wdump((original_auth));
 
-   set_authority( "alice"_n, config::active_name,  delegated_auth );
+   chain.set_authority( "alice"_n, config::active_name,  delegated_auth );
 
-   auto new_auth = control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
+   auto new_auth = chain.control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
    wdump((new_auth));
    BOOST_CHECK_EQUAL((new_auth == delegated_auth), true);
 
-   produce_block();
-   produce_block();
+   chain.produce_block();
+   chain.produce_block();
 
-   auto auth = control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
+   auto auth = chain.control->get_authorization_manager().get_permission({"alice"_n, config::active_name}).auth.to_authority();
    wdump((auth));
    BOOST_CHECK_EQUAL((new_auth == auth), true);
 
    /// execute nonce from alice signed by bob
-   auto trace = push_reqauth("alice"_n, {permission_level{"alice"_n, config::active_name}}, { get_private_key("bob"_n, "active") } );
+   auto trace = chain.push_reqauth("alice"_n, {permission_level{"alice"_n, config::active_name}}, { chain.get_private_key("bob"_n, "active") } );
 
-   produce_block();
+   chain.produce_block();
    //todoBOOST_REQUIRE_EQUAL(true, chain_has_transaction(trace->id));
 
 } FC_LOG_AND_RETHROW() }
 
 
 BOOST_AUTO_TEST_CASE(update_auths) {
 try {
-   validating_tester chain;
+   savanna_validating_tester chain;
+
```

Comment:
`chain.find` on line 120 below causes compilation failure when `BOOST_AUTO_TEST_CASE_TEMPLATE` is used. We test Savanna for now.

Reply:
Probably just need `chain.template find<permission_object, by_owner>(boost::make_tuple(name("alice"), name("owner")));`

Reply:
Thank! It worked.